test(editors/IED): Added some missing unit tests

Flurb opened the pull request.

JakobVogelsang reviewed (approved):
LGTM

JakobVogelsang commented:
One small remark. Added value to test file are always tagged with test: `test(somefile): added some thing`. I changed that hope this is alright.

Flurb commented:
> One small remark. Added value to test file are always tagged with test: `test(somefile): added some thing`. I changed that hope this is alright.

Got it! Next time, I will do it! :)
